core/templates/dev/head/forms/formBuilder.js

```diff
@@ -1358,6 +1378,17 @@ oppia.directive('schemaBasedFloatEditor', [function() {
           }
         };
 
+        if (typeof $scope.uiConfig() !== 'undefined' &&
+            $scope.uiConfig().slider === true) {
+          if (!$scope.localValue) {
+            $scope.localValue = 300.0;
+          }
+
+          $scope.$watch('localValue', function(newValue) {
+            imageComponentWidthService.set(newValue);
```

Comment:
Oh OK i understand. But I couldn't find a better way to assign a default value to the slider and also to link the width value to the slider.